examples/aligning_aia_with_eis_maps.py

```diff
@@ -0,0 +1,72 @@
+"""
+=====================
+Coaligning EIS to AIA
+=====================
+
+This example shows how to EISA data to AIA using cross-correlation which is implemented as the "match_template" method.
+"""
+
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.visualization import ImageNormalize, AsinhStretch
+from sunpy.net import Fido ,attrs as a
+import sunpy.map
+from sunkit_image.coalignment import coalign
+from sunkit_image.data.test import get_test_filepath
+
+###################################################################################
+# Firstly, let us acquire the EIS and AIA data we need for this example.
+
+eis_map = sunpy.map.Map(get_test_filepath("eis_20140108_095727.fe_12_195_119.2c-0.int.fits"))
```

Comment:
This file now lives here: https://github.com/sunpy/data/raw/main/sunkit-image/eis_20140108_095727.fe_12_195_119.2c-0.int.fits

We can pass this URL into map instead.

Reply:
This should also work for tests but we should create a scoped fixture.